Embrace Unity iOS Interface/EmbraceManager.swift

```diff
@@ -0,0 +1,494 @@
+import Foundation
+import OSLog
+import EmbraceIO
+import EmbraceCrash
+import EmbraceCommonInternal
+import EmbraceOTelInternal
+
+public class EmbraceManager: NSObject {
+    private var os_log = OSLog(subsystem: "Embrace", category: "UnityiOSNativeEmbraceManager")
+    private static var spanRepository = SpanRepository()
+    static func startNativeSDK(appId: String, appGroupId: String?, endpoints: (baseUrl: String, devBaseUrl: String, configBaseUrl: String)?) -> Bool {
+        do {
+            var embraceOptions: Embrace.Options {
+                var _crashReporter: CrashReporter? = EmbraceCrashReporter()
+                let _servicesBuilder = CaptureServiceBuilder().addDefaults()
+                _servicesBuilder.add(.pushNotification()) // Add the PushNotificationCaptureService by default for Unity
+                var _endpoints: Embrace.Endpoints? = nil;
+                
+                if let endpoints {
+                    _endpoints = Embrace.Endpoints(
+                        baseURL: endpoints.baseUrl,
+                        developmentBaseURL: endpoints.devBaseUrl,
+                        configBaseURL: endpoints.configBaseUrl)
+                }
+                
+                return .init(
+                    appId: appId,
+                    appGroupId: appGroupId,
+                    platform: .unity,
+                    endpoints: _endpoints,
+                    captureServices: _servicesBuilder.build(),
+                    crashReporter: _crashReporter
+                )
+            }
+            
+            try Embrace.setup(options: embraceOptions)
+                .start()
+            
+            return true
+        } catch let e {
+            print("Error starting Native Embrace SDK \(e.localizedDescription)")
+            return false
+        }
+        
+    }
+    
+    static func isStarted() -> Bool {
+        if let embraceStarted = Embrace.client?.started {
+            return embraceStarted
+        }
+        return false
+    }
+    
+    static func sdkVersion() -> String {
+        return Embrace.sdkVersion;
```

Comment:
Nitpick: swift doesn't require semicolons, and while it doesn't have any negative impact in this case, it's generally not used unless you're separating multiple statements on the same line.
```suggestion
        return Embrace.sdkVersion
```